server/gfmd/db_ops.h

```diff
@@ -311,4 +311,14 @@ struct db_ops {
 
 	gfarm_error_t (*fsngroup_modify)(gfarm_uint64_t,
 		struct db_fsngroup_modify_arg *);
+
```

Comment:
move the user_auth related functions to just after the (*user_load)() function,
because GfarmUserAuth is highly related to GfarmUser.

Reply:
commit [429eb3d](https://github.com/oss-tsukuba/gfarm/pull/1141/commits/429eb3daa151165a09117234d565149a2a1ac4e6) has not applied above change. Could you change it?